### ISSUE TITLE
Suggestions for France and the rest

### ISSUE THREAD

**Issue by Emp-Palpatine** [Victoria2CoE/concert-of-europe#31]
- First of all, flags. 
France should use the proper bourbon flag. 
I've an idea : why not create a new form of government that could be called "restorationist monarchy" or "executive monarchy", something like that?
It would help having France have its own flag until the more constitutionnalist Louis Philippe. Also, it could be used in Italy : Piemont-Sardinia could switch to the Tricolore too like it did in 1848, also Two sicilies in 1860. 

- France's revolutions should be dealed with multiple choices events. I could help with a few step-events leading to the revolution. At least, the player should have the choice to use force (contrary to what Charles X and Louis Philippe really did). I've quite a few ideas for the february revolution 1848 and the July Monarchy (like the role and possible death of Ferdinand Duke of Orléans)
I can draw a list of events to code with consequences if willing.

More to follow :)

**Comment by Emp-Palpatine** [comment]
New batch. I thought about all this at work.

- Parties.
Parties should be "regime  neutral" at least from the mid-19th century. I am frankly fed up with republican named parties coming to power in a staunch and solid French Kingdom or Empire. Got a file if you want, will upload it later.

- Regime characteristics. 
French successive regimes arent' portrayed very faithfully. I've altered my own install of victoria II to do this. It's especially true for the 2nd Empire.  Will post the suggestions shortly.

- As said before, France should have the white flag. I suggest this one 
https://upload.wikimedia.org/wikipedia/commons/thumb/6/64/Pavillon_Louis_XIV.png/240px-Pavillon_Louis_XIV.png
Actually, France lacked a law for terrestrial flag, so people used the plain white naval enseign. It was not a popular one and quickly variant with fleur de lys or lesser or greater coats of arms were used. This era press described in its article people waving white flag with the kingdom arms or the Lys during festivities or royal visits. The Greater coat of arms one is more the King's standart. The one I suggested with perhaps bigger coat of arms should do the trick.
Also closer to the regimental flags of the time
http://www.drapeaux.org/France/Drapeaux/Infanterie/1814/22_Ligne_1814_Revers.gif

And to what an eyewitness painter had drawn 
https://www.histoire-image.org/sites/default/lar7_cogniet_001f.jpg (left flag)

How? I suggest the new government method used in Death by Imperialism. Could be used for a lot of other countries (yes, thinking about you Italian State, Mexican Empire, Spain...)

Next time : restauration events for Louis XVIII and Charles X as well as what ifs if the Bourbons stay on the throne. 
Political parties corrected
Institutions corrected (I suppose the new government will be used).

**Comment by Emp-Palpatine** [comment]
All right, here are propositions for corrected parties. (1836 onwards) 
I think it could be possible to duplicate conservative parties on the church issue, between moralism and pluralism, it could follow the orleanism/legitimism with "droite opportuniste" being for pluralism (orleanist that historically accepted whoever was in power) and "parti de l'ordre" being the conservative monarchists, bonapartists and so on.  

party = {
	name = "FRA_reactionary" #Légitimiste
	start_date = 1830.1.1
	end_date = 1883.1.1

	ideology = reactionary

	economic_policy = state_capitalism
	trade_policy = protectionism
	religious_policy = moralism
	citizenship_policy = residency
	war_policy = pro_military
}

party = {
	name = "FRA_reactionary_2" #Action Française
	start_date = 1883.1.1
	end_date = 2000.1.1

	ideology = reactionary

	economic_policy = state_capitalism
	trade_policy = protectionism
	religious_policy = moralism
	citizenship_policy = residency
	war_policy = jingoism
}

party = {
	name = "FRA_liberal" #Libéraux
	start_date = 1830.1.1
	end_date = 1848.1.1

	ideology = liberal

	economic_policy = interventionism
	trade_policy = protectionism
	religious_policy = pluralism
	citizenship_policy = full_citizenship
	war_policy = anti_military
}

party = {
	name = "FRA_liberal_2" #Tiers parti
	start_date = 1848.1.1
	end_date = 1870.1.1
	
	ideology = liberal
	
	economic_policy = laissez_faire
	trade_policy = protectionism
	religious_policy = pluralism
	citizenship_policy = full_citizenship
	war_policy = anti_military	
}

party = {
	name = "FRA_liberal_3" #Modérés
	start_date = 1870.1.1
	end_date = 2000.1.1

	ideology = liberal

	economic_policy = laissez_faire
	trade_policy = protectionism
	religious_policy = secularized
	citizenship_policy = full_citizenship
	war_policy = anti_military
}

party = {
	name = "FRA_conservative" #Droite dynastique
	start_date = 1830.1.1
	end_date = 1848.1.1

	ideology = conservative

	economic_policy = laissez_faire
	trade_policy = protectionism
	religious_policy = pluralism
	citizenship_policy = limited_citizenship
	war_policy = pro_military
}

party = {
	name = "FRA_conservative_2" #Parti de l'ordre
	start_date = 1848.1.1
	end_date = 1870.1.1

	ideology = conservative

	economic_policy = interventionism
	trade_policy = free_trade
	religious_policy = moralism
	citizenship_policy = full_citizenship
	war_policy = jingoism
}

party = {
	name = "FRA_conservative_3" #Parti de l'ordre
	start_date = 1870.1.1
	end_date = 2000.1.1

	ideology = conservative

	economic_policy = interventionism
	trade_policy = protectionism
	religious_policy = pluralism
	citizenship_policy = full_citizenship
	war_policy = pro_military
}

party = {
	name = "FRA_socialist" #Républicain démocratique
	start_date = 1870.1.1
	end_date = 1905.1.1

	ideology = socialist

	economic_policy = state_capitalism
	trade_policy = protectionism
	religious_policy = secularized
	citizenship_policy = limited_citizenship
	war_policy = pacifism
}

party = {
	name = "FRA_socialist_2" #Possibilistes
	start_date = 1848.1.1
	end_date = 1870.1.1

	ideology = socialist

	economic_policy = state_capitalism
	trade_policy = protectionism
	religious_policy = secularized
	citizenship_policy = limited_citizenship
	war_policy = pacifism
}

party = {
	name = "FRA_socialist_3" #SFIO
	start_date = 1905.1.1
	end_date = 2000.1.1

	ideology = socialist

	economic_policy = state_capitalism
	trade_policy = protectionism
	religious_policy = secularized
	citizenship_policy = limited_citizenship
	war_policy = anti_military
}

party = {
	name = "FRA_anarcho_liberal" #Républicain radical
	start_date = 1830.1.1
	end_date = 2000.1.1

	ideology = anarcho_liberal

	economic_policy = laissez_faire
	trade_policy = free_trade
	religious_policy = secularized
	citizenship_policy = full_citizenship
	war_policy = pacifism
}

party = {
	name = "FRA_communist" #Parti Ouvrier Français
	start_date = 1848.1.1
	end_date = 2000.1.1

	ideology = communist

	economic_policy = planned_economy
	trade_policy = protectionism
	religious_policy = pro_atheism
	citizenship_policy = limited_citizenship
	war_policy = pacifism
}

party = {
	name = "FRA_fascist" #Fascists
	start_date = 1905.1.1
	end_date = 2000.1.1

	ideology = fascist

	economic_policy = state_capitalism
	trade_policy = protectionism
	religious_policy = moralism
	citizenship_policy = residency
	war_policy = jingoism
}

party = {
	name = "FRA_social_liberal" #Parti Radical
	start_date = 1890.1.1
	end_date = 2000.1.1

	ideology = social_liberal

	economic_policy = interventionism
	trade_policy = free_trade
	religious_policy = secularized
	citizenship_policy = full_citizenship
	war_policy = anti_military
}

**Comment by Emp-Palpatine** [comment]
Ok , time for some events for France's 1815/1830.
I can't code, so these are just ideas. If the team is interested, contact me and I'll write the description. These events could be multiple choices or not.


    Les Sergents de La Rochelle (septembre 1822)
    4 sergeants of the 45th regiment d'infanterie, a "politically unsure unit" founded their own carbonari cell. Denounced, they were trialed, condamned to death and guillotined.
    Suggested effects: + x% of liberals among middle and low classes (low %), slight increase of militancy, big increase of consciousness

    mort de Louis XVIII (16 septembre 1824)

Even if he was disliked in liberal circles and at first in the military, Louis XVIII became quite popular outside Paris and in the countryside, especially since his reign was one of peace and reconstruction. His death meant the ascendance of his brother as Charles X
suggested effects : - adding a flag "Charles X" that could make revolution easier
Some token prestige (well, first ruler to die peacefully since 1774 and first peacefull and constitutionnal transition since 1774, and last one until the early 20th (!!) )

    Le sacre

29 mai 1825: Sacre de Charles X
Louis XVIII wasn't coronnated, both for physical and political reasons. Despite his age, Charles X was still in excellent health and way more attached to the Old Regime pump. Popular upon his accession to the throne, the sacre was nonetheless one of the first event that upsetted the liberals.
Suggested effects : prestige!
Conservative & reactionnary : decrease of militancy and consciousness
Rise of consciousness for liberals (worried by the evolution of the regime)

    Loi du milliard des émigrés (spring 1825)

This law's goal was to pay back anyone that lost property following the revolution's nationalisations. It angered the new elites, that became rich and influent during those days, thanks to their new properties...
Suggested effects : cost of cash (the money was taken from public spending), increase of consciousness and militancy for liberal and middle classes.

    Loi sur le sacrilège ( spring 1825)

This law condemned to death any desacration of sacred objects or of the Saintes Espèces (don't know how to translate these, that is consecrated bread and wine). It was never applied but still triggered and uproar among liberals and easily anti-clerical french
Suggested effects : rise in consciousness & militancy among liberals, especially among artisans, bureaucrats and craftmen.

    Dissolution de la garde nationale

Spring 1827
French Prime minister Villèle was booed during a review of the parisian national guard and then the King was welcomed icely. The government choosed to dissolve the parisian section of the guard. Liberals were furious.
Suggest effects: rise in militancy among middle class in Paris.

    Gouvernement ultra

1829: Charles X unilaterally choose a new cabinet, despite the popularity of the moderate cabinet that successed Villèle...

    L'adresse des 221

March 1830 : an adress is presented to the King by 221 members of parliament as the ultra Polignac do not have any majority in the lower house. The parliament can't chase the prime minister, but very respectfully asked the king to reconsider its choice and to make it more coherent with both houses views.

    Les ordonnances de Saint Cloud

In order to break the deadlock, Charles X dissolved the lower house but the liberal won the (wealthed) vote.
He then tried to change won the struggle more violently:
- 1st ordinance suspensed the freedom of press (so, State press)
- 2nd one dissolved the parliament (since then, it's forbidden in France's constitutionnal law to dissolve twice within the same year)
- 3rd one restricted the vote (so... landed ?)

Regarding the revolution, I propose either a dynamic one, thanks to the Charles X flag : as soon as the militancy top a certain level or controll of Paris is lost to rebel, the "Trois Glorieuses" could be triggered. Also, press rights should be triggers as the main thing that pushed liberal in the street was the censorship thing.
Or they could trigger only if Paris is uncontrolled.
Or by event.
Any way, I propose the said choices upon the Trois Glorieuses:

A/ Historical one : Charles X abdicate, his son renounce, Louis-Philippe designated as regent and "forget" to say it is in the name of Henry V...
effect: switching tag or switching government, beginning of the July Monarchy and of course tricolore flag.

B/ Stay and fight! Charles X retreat in the proximity of Paris and will use force to re-take paris (was his first plan but Marshall Marmont was a defeatist) as the regime wasn't contested in the countryside.
effect: obvious hit of militancy for liberals.

C / Abdication for Henry V, Louis-Philippe does not usurp the throne.
Charles X abdicate and his son too (unpopular and stupid, it was quite obvious back than that the next king would be Henry), as historically. Louis-Philippe this time anounce his regence but do not forget to say it's in the name of the young Henry.
Effect: some reforms -will say-, France is still Bourbon, conservative decrease of militancy & consciousness. No effect for liberals (or slight decrease? The hated Charles X and his son are gone)

D/ Republic it is!
Historically, the liberals that triggered the revolution were no republicans and wanted reforms or a change of dynasty. They were quickly frightened of a renewed 1793 by the Parisians radical artisans. Hence why Lafayette embraced Louis Philippe this fast.
But now, time for a radical republic!
Effect : radical republic, obvious civil war against a mainly conservative France and/or foreign intervention.

**Comment by Emp-Palpatine** [comment]
Here are some suggestions for more historical french governments.


1821 start :

government = prussian_constitutionalism
slavery = yes_slavery
upper_house_composition = appointed
vote_franschise = weighted wealth
public_meetings = no_meeting
press_rights = censored_press (debatable)
trade_unions = no_trade_unions
minorities_reform = limited
immigration_policy = quota_immigration
conscription = no_draft
voting_system = first_past_the_post
political_parties = gerrymandering



1830 july monarchy

government = HM's government (debatable)
slavery = yes_slavery
upper_house_composition = appointed
vote_franschise = weighted wealth/wealth voting (debatable)
public_meetings = allowed (debatable)
press_rights = free press (debatable)
trade_unions = no_trade_unions
minorities_reform = limited
immigration_policy = quota_immigration
conscription = no_draft
voting_system = first_past_the_post
political_parties = gerrymandering

The fact is the regime evolved toward censorship and repression after the first couple of years, reintroducing censorship and severly crushing freedom of assembly because of riots. This could be treated by events by putting back no meetings and censorship
Also, Louis Philippe rule was quite discreet and he relied on his Prime ministers... So, HM's government is quite ok.
The franchise was quite extended by the 1830 revision of the Constitutionnal charter, so Wealth could do.


1848 2nd republic :

government = Constitutionnal republic (or whatever the name is)
slavery = yes_slavery (debatable, is it treated by decision/event?)
upper_house_composition = according to population
vote_franschise = universal voting
public_meetings = allowed
press_rights = free press
trade_unions = no_trade_unions
minorities_reform = limited
immigration_policy = quota_immigration
conscription = no_draft
voting_system = first_past_the_post
political_parties = non secret ballots


2nd Empire :

government = Prussian constitutionnalism (or whatever the name is)
slavery = no_slavery (should be abolished by then ?)
upper_house_composition = appointed
vote_franschise = universal voting
public_meetings = no_meeting
press_rights = censored press/state press (debatable)
trade_unions = no_trade_unions
minorities_reform = limited
immigration_policy = quota_immigration
conscription = no_draft
voting_system = first_past_the_post
political_parties = harassment/underground parties (debatable)


3rd republic :

government = Constitutionnal republic (or whatever the name is)
slavery = no_slavery (should be abolished by then ?)
upper_house_composition = two per state
vote_franschise = universal voting
public_meetings = no_meeting (debatable)
press_rights = free_press
trade_unions = all allowed (debatable, 2nd Empire did legalize them)
minorities_reform = limited (debatable, 3rd republic was horrible against local languages and such)
immigration_policy = quota_immigration
conscription = one year draft
voting_system = proportional
political_parties = non secret ballots


Possible Legitimist restauration :

government = Prussian constitutionnalism (or whatever the name is)
slavery = no_slavery (should be abolished by then )
upper_house_composition = appointed/two per state (Henry's ideas were quite federalist)
vote_franschise = universal voting
public_meetings = no_meeting (don't know for this one)
press_rights = censored press (debatable)
trade_unions = non socialist allowed (should be legalized by 2nd Empire)
minorities_reform = protected (Henry was quite clear on this)
immigration_policy = quota_immigration
conscription = no_draft
voting_system = first_past_the_post
political_parties = non secret ballots

This one is tricky. Henry project was quite clear for the institutions : a monarchy with a written constitution, high decentralisation of power towards provinces, two houses, universal voting. Was less obvious for things such as press right. I must check my books.



Possible Orleanist restauration :


government = Constitutionnal republic (or whatever the name is)
slavery = no_slavery (should be abolished by then ?)
upper_house_composition = two per state
vote_franschise = universal voting
public_meetings = no_meeting (debatable)
press_rights = free_press
trade_unions = all allowed (debatable, 2nd Empire did legalize them)
minorities_reform = limited (debatable)
immigration_policy = quota_immigration
conscription = one year draft
voting_system = proportional
political_parties = non secret ballots

Basically a crowned 3rd republic: this was the plan and the constitutionnal laws were written with a constitutionnal monarchy in mind.

**Comment by GeneralVonGelre** [comment]
Party names are taken care of
